maplibre_gl_web/lib/src/mapbox_web_gl_platform.dart

```diff
@@ -881,6 +881,24 @@ class MaplibreMapController extends MapLibreGlPlatform
         enableInteraction: enableInteraction);
   }
 
+  Future<void> setLayerProperties(
+      String layerId, Map<String, dynamic> properties) async {
+    for (final entry in properties.entries) {
+      // Very hacky: because we don't know if the property is a layout
+      // or paint property, we try to set it as both.
+      try {
+        _map.setLayoutProperty(layerId, entry.key, entry.value);
+      } catch (e) {
+        print('Caught exception (usually safe to ignore): $e');
+      }
+      try {
+        _map.setPaintProperty(layerId, entry.key, entry.value);
+      } catch (e) {
+        print('Caught exception (usually safe to ignore): $e');
+      }
```

Comment:
```suggestion
      try {
        _map.setLayoutProperty(layerId, entry.key, entry.value);
        _map.setPaintProperty(layerId, entry.key, entry.value);
      } catch (e) {
        print('Caught exception (usually safe to ignore): $e');
      }
```

Reply:
@stefanschaller What do you think, is it fine to keep the code as is with that explanation?